checks/webhook.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2022 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package checks
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+
+	"github.com/ossf/scorecard/v4/checker"
+	"github.com/ossf/scorecard/v4/checks/raw"
+	sce "github.com/ossf/scorecard/v4/errors"
+)
+
+const (
+	// CheckWebHooks is the registered name for WebHooks.
+	CheckWebHooks = "Webhooks"
+)
+
+//nolint:gochecknoinits
+func init() {
```

Comment:
overall PR is almost read to merge.

There's one thing you need to update (it may be confusing). The file `checks/webhook.go` need not contain much logic. 

The way the implementation works is the following: 
1. `checks/webhook.go` should call `checks/raw/webhook.go` to collect the data. You already have this file implemented, so I don't expect further changes there
2. `checks/webhook.go`  verifies data was retrieved properly, then calls `checks/evaluation/webhook.go`. See binary-artifact code for example https://github.com/ossf/scorecard/blob/main/checks/binary_artifact.go.
3. `checks/evaluation/webhook.go` reads the results and uses its logic to decide what should be logged, using `Warn`, `Info`, etc. The "evaluation" is essentially where decision takes place regarding what to do with the results retrieved by `checks/evaluation/webhook.go`. The logic you have in this file should basically be moved to `checks/evaluation/webhook.go` instead. (see https://github.com/ossf/scorecard/blob/main/checks/evaluation/binary_artifact.go as example)
4. 
Does this clarify things?

Reply:
thanks for the feedback
made the changes PTAL